sympy/sets/tests/test_fancysets.py

```diff
@@ -198,8 +198,10 @@ def test_Complex():
     assert -I in S.Complexes
     assert sqrt(-1) in S.Complexes
     assert S.Complexes.intersect(S.Reals) == S.Reals
-    # assert S.Complexes.union(S.Reals) == S.Complexes
+    assert S.Complexes.union(S.Reals) == S.Complexes
```

Comment:
This i think i myself commented it out, since of the `hash` problem described by Aaron previously, seems to be working fine now since of PR #10420 (though not sure)